sdk/python/requirements/py3.10-requirements.txt

```diff
@@ -20,32 +20,38 @@ charset-normalizer==3.3.2
     # via requests
 click==8.1.7
     # via
+    #   feast (setup.py)
```

Comment:
just curious, what command did you run to lock these files? I'm not sure why this `feast (setup.py)` lines are being added here...

Reply:
`make lock-python-dependencies PYTHON=3.xx`

Reply:
+1